### DISCUSSION TITLE
Linux Distribution via AppImage file

### DISCUSSION THREAD

**Discussion by ghost** [mullvad/mullvadvpn-app#9066] [Feature requests & Ideas]
There are quite a few distros when it comes to Linux. Please also provide an AppImage file for release. Thank you.

**Comment by who-biz** [comment]
@wuzhe if you are still in need of an AppImage compilation, I have modified GUI distribution files to create one for non-rpm systems.  You can find it, and supporting info here: https://github.com/who-biz/mullvadvpn-app-appimage

**Comment by who-biz** [comment]
Hi - I see this was converted to a discussion.  Let me know if there is any way I can help. This would enable me to run Mullvad on more systems without worrying about missing critical security updates, when maintaining it myself.

**Reply by MarkusPettersson98** [reply]
Hi @who-biz, we are in the wake of migrating feature suggestions from issue to discussions so we might be slower than usual to respond. Sorry about that!

Personally I think it would be a net-positive to distribute the app as an AppImage if possible as the app would reach more users. With that said, there is no plan to distribute the app as an AppImage. I'll bring up the discussion internally and see what other people's opinions are!

**Reply by who-biz** [reply]
Great. Let me know if there's any way I can contribute meaningfully. It would be great having more confidence that I'm not breaking things, when packaging in this manner. (via  the eyes of your official maintainers)

Ultimately, my concerns catalyzed me to switch back to Debian/Ubuntu. Mullvad is absolutely critical for my privacy expectations and needs. It was - in fact - the single deciding factor for me. 

I'm sure there are other users with similar tradeoff assessments in this boat with me.

**Comment by eylenburg** [comment]
I think an Appimage would be a good solution for non-supported and immutable Linux distros, if a Flatpak version is too hard to do
